Tests/FluentAssertions.Specs/Equivalency/BasicEquivalencySpecs.cs

```diff
@@ -596,6 +596,56 @@ public void When_the_graph_contains_guids_it_should_properly_format_them()
             act.Should().Throw<XunitException>().WithMessage("Expected item[0].Id to be *-*, but found *-*");
         }
 
+        [Fact]
+        public void When_defining_comparing_by_members_with_callback_that_returns_false_then_the_normal_equality_strategy_process_should_be_used()
+        {
+            // Arrange
+            var subject = new ClassWithValueSemanticsOnSingleProperty
+            {
+                Key = "key",
+                NestedProperty = "SomeValue"
+            };
+
+            var expected = new ClassWithValueSemanticsOnSingleProperty
+            {
+                Key = "key",
+                NestedProperty = "OtherValue"
+            };
+
+            // Act
+            Action act = () => subject.Should().BeEquivalentTo(expected, options => options
+                .ComparingByMembers(t => false));
+
+            // Assert
+            act.Should().NotThrow();
+        }
+
+        [Fact]
+        public void When_defining_comparing_by_members_with_callback_it_should_use_force_members_strategy_if_a_callback_returning_true_is_found()
```

Comment:
🤔 So is there some implicit or explcit order? Would be nice to capture that in a functional test name.